src/components/Screen.tsx

```diff
@@ -17,11 +17,19 @@ import {
 import ScreenNativeComponent from '../fabric/ScreenNativeComponent';
 import ModalScreenNativeComponent from '../fabric/ModalScreenNativeComponent';
 
-export const NativeScreen: React.ComponentType<ScreenProps> =
-  ScreenNativeComponent as React.ComponentType<ScreenProps>;
+type NativeScreenProps = Omit<
+  ScreenProps,
+  'sheetInitialDetentIndex' | 'sheetLargestUndimmedDetentIndex'
+> & {
+  sheetInitialDetent: number;
+  sheetLargestUndimmedDetent: number;
+};
```

Comment:
Had to do this to avoid renaming these props on native side (don't want to spend time right now on it, we will be able to change it later once this is done: https://github.com/orgs/software-mansion/projects/3/views/1?pane=issue&itemId=82038810)